script/Ciel/Day 14/14_01B_01_CIEL14B_1C.txt

```diff
@@ -709,7 +807,10 @@ Senpai nods in silence, runs toward the stairs, and stops as if suddenly remembe
 [sha:2e7a60de43208b1f3873046462c2f8017de2502d]{
 -- Page 143, Offset 38728.
 -- 「はい。……その、明日になったら街に遊びに行きましょう。乾くんと三人で、この前のやり直しをするんです」
-"Yes... Um, we should go out to the city and play tomorrow. Just as we planned with Inui-kun and the two of us."
+// Merged version of 37878/37879/37880
+"Yes.
+ Um... once everything is over, we should go enjoy ourselves in the city together.
+ We can invite Inui-kun and do a re-take of the plans we made before."
```

Comment:
From 37878/37879/37880
>Yes.
 Um... once everything is over, we should go hang out in the city together.
We can invite Inui-kun and do a re-take of the plans we made before.